web/src/components/MapView.tsx

```diff
@@ -1,21 +1,61 @@
-import React, { useState } from 'react'
-import ReactMapGL from 'react-map-gl'
+import React, { ReactElement, useEffect, useState } from 'react'
+import ReactMapGL, { Layer, LayerProps, MapRef, Source } from 'react-map-gl'
 import styled from 'styled-components'
 import 'maplibre-gl/dist/maplibre-gl.css'
-
-import { defaultViewportConfig, mapConfig } from 'api-client'
+import { customMarker, defaultViewportConfig, mapConfig } from 'api-client'
+import { FeatureCollection } from 'geojson'
 
 const MapContainer = styled.div`
   display: flex;
   justify-content: center;
 `
 
-const MapView: React.FunctionComponent = () => {
+const textOffsetY = 1.75
+const layerStyle: LayerProps = {
+  id: 'point',
+  type: 'symbol',
+  source: 'point',
+  layout: {
+    'icon-image': ['get', 'symbol'],
+    'text-field': ['get', 'title'],
+    'text-font': ['Roboto Regular'],
+    'text-offset': [0, textOffsetY],
+    'text-anchor': 'top',
+    'text-size': 12
+  },
+  paint: {
+    'icon-color': 'blue'
+  }
+}
+
+interface MapViewProps {
+  featureCollection: FeatureCollection
+}
+
+const MapView: React.FunctionComponent<MapViewProps> = (props: MapViewProps): ReactElement => {
   const [viewport, setViewport] = useState(defaultViewportConfig)
+  const { featureCollection } = props
+
+  const mapRef = React.useRef<MapRef>(null)
+  useEffect(() => {
+    const map = mapRef?.current?.getMap()
+    map?.loadImage(customMarker.iconSrc, (error: Error, image: ImageBitmap) => {
+      if (error) {
+        throw new Error('Image can not be loaded')
+      }
+      if (!map.hasImage(customMarker.name)) {
+        map.addImage(customMarker.name, image, { sdf: true })
```

Comment:
What does `sdf` stand for?

Reply:
sdf is a image type that enabled the loaded image to be colored